hw/opentitan/ot_csrng.c

```diff
@@ -934,6 +935,7 @@ static void ot_csrng_handle_enable(OtCSRNGState *s)
             }
         }
         s->enabled = false;
+        s->regs[R_SW_CMD_STS] &= R_SW_CMD_STS_CMD_RDY_MASK;
```

Comment:
I'm not sure I understand this, is it supposed to be `~R_SW_CMD_STS_CMD_RDY_MASK`?

Reply:
Yes, the `~` was missing. More broadly, I'm a little worried about that part not having good test coverage.